src/index.ts

```diff
@@ -80,12 +80,12 @@ export async function readResources(
       "{language}",
       language
     );
-    if((await args.$fs.stat(`${resourcePath.replace("/*.json", "")}`)).isFile()) {
+    try {
       const json = JSON.parse((await args.$fs.readFile(resourcePath, "utf-8")) as string)
```

Comment:
Will also break :D I hopefully merge `@inlang-git/fs` today. If that happens, you can be sure that the API works after upding `@inlang/core` here